cyfinfaza/sc4hfair-sveltekit
# get ready for 2023

* fixes #1 
* update to latest sveltekit & other libraries

todo:
* ensure all supabase stuff is working
* make sure caching isn't breaking anything
